benchmark/micro/json/RealNest/3_select_avg_from_unnested_structure.benchmark

```diff
@@ -0,0 +1,21 @@
+# name: benchmark/micro/json/RealNest/3_select_avg_from_unnested_structure.benchmark
+# description: select average from unnested structure Jet and group by, having
+# group: [RealNest]
+
+name select_avg_from_unnested_structure
+group real_nest
+
+require json
+
+load
+CREATE TABLE Run2012B_SingleMu AS SELECT * FROM read_json('duckdb_benchmark_data/hep-adl-ethz-Run2012B_SingleMu/data.jsonl');
+
+run
+SELECT puId, avg(pt), avg(eta), avg(phi), avg(mass), avg(btag)
+  FROM (
+      SELECT UNNEST(Jet, recursive:=true) AS unnested_jet
```

Comment:
Is `Jet` a list? Are `mass` and `eta` nested types?

Reply:
`Jet` is a `list` type, yeah.
 `mass` and `eta` are nested as well as `pt`, `btag`, `phi`

Reply:
Great, then this also benchmarks aggregating nested values 👍